frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -70,6 +70,8 @@ export const cleanFunnelParams = (filters: FilterType): FilterType => {
         ...(filters.funnel_step ? { funnel_step: filters.funnel_step } : {}),
         ...(filters.funnel_viz_type ? { funnel_viz_type: filters.funnel_viz_type } : {}),
         ...(filters.funnel_step ? { funnel_to_step: filters.funnel_step } : {}),
+        ...(filters.entrance_period_start ? { entrance_period_start: filters.entrance_period_start } : {}),
```

Comment:
Hm I'm kind of confused by what `entrance_period_start` means. Maybe a comment here or the corresponding type would be helpful.